### PR TITLE
add documentation about truenas server

### PR DESCRIPTION
**Pull request by larsks**

### CODE REVIEW

File: docs/source/operations/truenas-storage-for-zero-cluster.md [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+1. Name: da0
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r1w1e2
+   descr: LSI MR SAS 6G 1GB
+   lunid: 6003005701e387502117a99e129b7f83
+   ident: 00837f9b129ea917215087e301570003
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
```
Comment (naved001) [inline docs/source/operations/truenas-storage-for-zero-cluster.md:76]: what is the importance of having these specific bits of information (ident, lunid, fwsecorts etc) about all these drives? I don't see any useful information here.

File: docs/source/operations/truenas-storage-for-zero-cluster.md [hunk 2]
```diff
@@ -0,0 +1,645 @@
+# TrueNAS NFS Storage for Zero OpenShift Cluster
+
+## Overview
+
+We repurposed an recently retired Fujitsu server, `10.0.17.8`, to
```
Comment (naved001) [inline docs/source/operations/truenas-storage-for-zero-cluster.md:5]: typo

*a* recently retired...

File: docs/source/operations/truenas-storage-for-zero-cluster.md [hunk 3]
```diff
@@ -0,0 +1,645 @@
...
+   ident: 006cbfe31d76dfe61bc05c2709b00506
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
+
+```
+
+## Zpool configuration
+
+The drives are configured as a set of three RAIDZ2 vdevs with two hot
```
Comment (naved001) [inline docs/source/operations/truenas-storage-for-zero-cluster.md:576]: looks like there are 4 sets not three? unless I am not understanding it correctly.

**Comment by larsks** [comment]
Good call on the reviews; I've updated things based on your comments.

**Review by naved001 — APPROVED** [review]

**Pull request merged**